_includes/main_nav_tr.html

```diff
@@ -1,6 +1,8 @@
  <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top pl-md-5 pr-md-5 pl-sm-3 pr-0">
 
-    <a class="navbar-brand" href="{{ site.url }}/{{ page.lang }}/"><img src="{{ site.url }}/images/bisq-logo.svg" height="30"/></a>
+   <div>
```

Comment:
Any particular reason for adding this div element?

Reply:
@m52go They were vestiges of when the warning tool was placed in the navbar.